Commands/Public/imgur.js

```diff
@@ -0,0 +1,120 @@
+const imgur = require("imgur");
+const { IsURL } = require("../../Modules/Utils/index");
+const { tokens: { imgurClientID } } = require("../../Configurations/auth");
+
+imgur.setClientId(imgurClientID);
+
+module.exports = async ({ Constants: { Colors, Text } }, documents, msg, commandData) => {
+	if (msg.suffix === "credits" && configJSON.maintainers.includes(msg.author.id)) {
+		const c = await imgur.getCredits();
+		return msg.send({
+			embed: {
+				color: Colors.INFO,
+				title: "Imgur Credits Info",
+				description: require("util").inspect(c.data)
+					.slice(2, -2)
+					.replace(/ {2}|,/g, ""),
+			},
+		});
+	}
+	let filesToUpload = [];
+	if (msg.attachments.size) {
+		[...msg.attachments.values()].forEach(a => {
+			filesToUpload.push(a.url);
+		});
+	}
+	if (msg.suffix) {
+		filesToUpload = filesToUpload.concat(
+			msg.suffix.split(" ")
+				.trimAll()
+				.filter(IsURL));
+	}
+	if (!filesToUpload.length) {
+		return msg.send({
+			embed: {
+				color: Colors.INVALID,
+				title: `What should I upload? 😕`,
+				description: Text.INVALID_USAGE(commandData, msg.guild.commandPrefix),
+				footer: {
+					text: `Protip: You can use multiple URLs (separated by spaces) or attachments to upload them in one album!`,
+				},
+			},
+		});
+	}
+	await msg.send({
+		embed: {
+			color: Colors.INFO,
+			title: "Uploading your images ⌛",
+			description: "Please stand by...",
+		},
+	});
+
+	let results = [];
+	let albumLink;
+	try {
+		if (filesToUpload.length === 1) {
+			results.push(await imgur.uploadUrl(filesToUpload[0]));
+		} else {
+			const album = await imgur.createAlbum();
+			albumLink = `https://imgur.com/a/${album.data.id}`;
+			for (const url of filesToUpload) {
+				results.push(await imgur.uploadUrl(url, album.data.deletehash));
+			}
+		}
+	} catch (err) {
+		winston.debug(`Failed to upload image to Imgur`, { svrid: msg.guild.id, chid: msg.channel.id, usrid: msg.author.id }, err);
+		switch (err.status) {
+			case 400:
+				if (err.message === "File is over the size limit") {
+					return msg.send({
+						embed: {
+							color: Colors.SOFT_ERR,
+							title: "This file is huge! 💥",
+							description: "You can only upload files with up to 10MB each! Blame Imgur for that limit.",
```

Comment:
Remove the cheeky "Blame Imgur for that limit", we have enough sass elsewhere already!

Reply:
And the old bot was littered with sass and emojis. Did we suddenly change directions?

Reply:
You can never have enough sass :mmLOL: